compiler/prelude/prelude.go

```diff
@@ -99,19 +99,19 @@ var $ifaceMethodExpr = function(name) {
 };
 
 var $subslice = function(slice, low, high, max) {
+  if (high === undefined) {
+    high = slice.$length;
+  }
+  if (max === undefined) {
+    max = slice.$capacity
```

Comment:
This line is missing a semicolon at the end.

Reply:
Done.